make-all.sh

```diff
@@ -232,18 +265,22 @@ while true ; do
         _GENERATOR="-G Ninja"
         shift 1 ;;
     --no-build)
-        _DO_BUILD=0
+        _do_build=0
         shift ;;
     --no-krnlmon)
-        _WITH_KRNLMON=FALSE
+        _WITH_KRNLMON=OFF
         shift ;;
     --no-ovs)
-        _WITH_OVS=0
-        _WITH_OVSP4RT=FALSE
+        _WITH_OVSP4RT=OFF
+        _with_ovsp4rt=0
         shift ;;
     --no-rpath)
         _RPATH=OFF
         shift ;;
+    --p4ovs)
+        # convert to uppercase
+        _P4OVS_MODE=${2^^}
```

Comment:
Not familiar with what the `^^` does

Reply:
It converts a string to uppercase. `,,` converts it to lowercase. 😎